examples/dt/dcn/values.yaml.j2

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+data:
+    customDataplanService:
+      name: nova-custom-ceph-{{ _az }}
+    nodeset_name: {{ _group_name }}-edpm
+    ceph_conf:
+{% for _file in _ceph_files %}
+        {{ _file | basename }}: {{ lookup('file', _file, rstrip=False) | b64encode }}
+{% endfor %}
+    nodeset:
+        services:
+        - bootstrap
+        - configure-network
+        - validate-network
+        - install-os
+        - ceph-hci-pre
+        - configure-os
+        - ssh-known-hosts
+        - run-os
+        - reboot-os
+        - install-certs
+        - ceph-client
+        - ovn
+        - neutron-metadata
+        - libvirt
+        - nova-custom-ceph-{{ _az }}
+    nova:
+        ceph:
+            conf: |
+                [libvirt]
+                images_type=rbd
+                images_rbd_pool=vms
+                images_rbd_ceph_conf=/etc/ceph/{{ cifmw_ceph_client_cluster }}.conf
+                images_rbd_glance_store_name=default_backend
+                images_rbd_glance_copy_poll_interval=15
+                images_rbd_glance_copy_timeout=600
+                rbd_user=openstack
+                rbd_secret_uuid={{ cifmw_ceph_client_fsid }}
```

Comment:
We should add some more parameters to be set, the glance endpoint and the cross_az_attach parameter, so It could look like:
```

    [glance]
    endpoint_override = https://glance-{{ _az }}-internal.openstack.svc:9292
    valid_interfaces = internal
    [cinder]
    cross_az_attach = False
    catalog_info = volumev3:cinderv3:internalURL
```
Additionally to the [libvirt] section

Reply:
Done